blocks_vertical/data.js

```diff
@@ -486,6 +486,40 @@ Blockly.Extensions.registerMixin('contextMenu_getVariableBlock',
   Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_VARIABLE_MIXIN);
 
 /**
+ * Mixin to add a context menu for a data_list block.  It adds one item for
+ * each list defined on the workspace.
+ * @mixin
+ * @augments Blockly.Block
+ * @package
+ * @readonly
+ */
+Blockly.Constants.Data.CUSTOM_CONTEXT_MENU_GET_LIST_MIXIN = {
```

Comment:
Whoops, sorry my comment wasn't clear. This part should be future work (following Rachel's suggestion below) but since it's not used in this CL you don't need to add it yet.